src/dodal/beamlines/i22.py

```diff
@@ -309,3 +310,16 @@ def linkam(
         wait_for_connection,
         fake_with_ophyd_sim,
     )
+
+
+@skip_device
```

Comment:
Why is this device skipped for i22?

Reply:
I didn't want interfere with anything, re-enabled in 6d5ae3fb866bdad8de27966fc11605b6c41856cd